pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -2647,7 +2647,7 @@ public void startSendOperation(Producer producer, int msgSize, int numMessages)
             // client connection, possibly shared between multiple producers
             ctx.channel().config().setAutoRead(false);
             recordRateLimitMetrics(producers);
-            autoReadDisabledRateLimiting = isPublishRateExceeded;
+            autoReadDisabledRateLimiting = true;
```

Comment:
This change does not follow the original design of `autoReadDisabledRateLimiting`.
Can you provide more info about the effect when `pendingSendRequest == maxPendingSendRequests` and `autoReadDisabledRateLimiting` is false.

Reply:
ok，maybe I misunderstand the design